frontend/.gitignore

```diff
@@ -43,3 +43,5 @@ testem.log
 # System Files
 .DS_Store
 Thumbs.db
+
+package-lock.json
```

Comment:
We've already set in package-lock=false .npmrc

So we don't need these changes
(Hopefully that no one will commit this file again)